client/src/middlewares/loadLevelInstance.js

```diff
@@ -62,6 +63,11 @@ const loadLevelInstance = (store) => (next) => (action) => {
           }
           if (!instanceAddress) {
             showErr(strings.transactionNoLogsMessage)
+          } else {
```

Comment:
I would add a flag true/false to know if the verification shuld be triggered or not. Some contracts don't need it. This flag should go in the gamedata file.

Reply:
Yeah, I've added that check inside the function [here](https://github.com/OjusWiZard/ethernaut/blob/contract-verification/client/src/utils/contractutil.js#L175). That's because I tried make minimum changes in the middleware and keep in clean. We can move it here if you want?